ansible/library/port_alias.py

```diff
@@ -48,7 +48,8 @@
 ### Here are the expectation of files of device port_config.ini located, in case changed please modify it here
 FILE_PATH = '/usr/share/sonic/device'
 PORTMAP_FILE = 'port_config.ini'
-ALLOWED_HEADER = ['name', 'lanes', 'alias', 'index', 'asic_port_name', 'role', 'speed']
+ALLOWED_HEADER = ['name', 'lanes', 'alias', 'index', 'asic_port_name', 'role', 'speed',
+                  'coreid', 'coreportid', 'numvoq']
```

Comment:
Adding extra fields 'coreid', 'coreportid', and 'numvoq' in port_config.ini will require updating port_config.ini for all platforms across all vendors. Has this been discussed and approved in sonic chassis subgroup.  

Would sonic-buildimage be compatible with these added fields in port_config.ini.

Reply:
When I emailed sonic chassis subgroup there was no opposition. 

After looking at asic_port_name usage only in multi-asic-vs port_config.ini file,
(e.g device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/2/port_config.ini) I wanted to follow similar logic of adding coreid, coreportid, numvoq only to the applicable VoQ sku's port_config.ini and it shouldn't be required in all skus.

Reply:
As long as compatibility is handled in sonic-buildimage - it is good.